django/contrib/auth/validators.py

```diff
@@ -23,3 +24,48 @@ class UnicodeUsernameValidator(validators.RegexValidator):
         'numbers, and @/./+/-/_ characters.'
     )
     flags = 0
+
+
+class ValidatorList:
+    '''
+    The validator list helps manage validators of defined type.
+    '''
+    def __init__(self, validator_type):
+        self.validators = []
+        self.validator_type = validator_type
+    
```

Comment:
flake8 is telling you you have whitespace in an empty line here. To view all the times flake8 found this, go to your pull request, view the automatic checks at the bottom, select Details next to the flake8 test, click Console, scroll bottom where you will see file names and line numbers and the specific problem.